app/operations/story_operations/read_all.rb

```diff
@@ -32,12 +32,12 @@ def project_iterations
 
     def active_stories
       @active_stories ||= begin
-        project
-          .stories
-          .with_dependencies
-          .where("state != 'accepted' OR accepted_at >= ?", current_iteration_start)
-          .order('updated_at DESC')
-      end
+          project
+            .stories
+            .with_dependencies
+            .where("state != 'accepted' OR accepted_at >= ?", current_iteration_start)
```

Comment:
I see we duplicate this logic in this PR. What do you think about creating a scope for that?